apps/provisioning/pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -511,12 +511,14 @@ type ResourceList struct {
 }
 
 type ResourceListItem struct {
```

Comment:
ResourceListItem's use intersects with JobResourceResult and I can't hack my way around returning Kind values in Resource field just cuz list UX doesn't use the plural in the table format.

The Lookup table from GroupKind to ResourceInfo or GroupResource to ResourceInfo behaves as expected only when we have the usage of resource and kind aligned everywhere.

The TL; DR on lookups is that they are necessary. Unistore doesns't have a way of returning Kind at all. It only returns "resource" in both List and Count operations. It confusingly has a `Kind` on Count results but its actually ManagerKind in practice (so "repo", "terraform", "plugins" etc.).